tests/unit/_experimental/test_plots.py

```diff
@@ -0,0 +1,1229 @@
+                    ],
+                    unmatched_gt=[
+                        (LabeledBoundingBox((1, 1), (2, 2), "a"), None),
+                    ],
+                    unmatched_inf=[
+                        ScoredLabeledBoundingBox((8, 8), (9, 9), "a", 0),
+                    ],
+                ),
+                MulticlassInferenceMatches(
+                    matched=[
+                        (LabeledBoundingBox((1, 1), (4, 4), "b"), ScoredLabeledBoundingBox((1, 1), (4, 4), "b", 0)),
+                        (LabeledBoundingBox((2, 2), (7, 7), "b"), ScoredLabeledBoundingBox((2, 2), (7, 7), "b", 0)),
+                    ],
+                    unmatched_gt=[
+                        (LabeledBoundingBox((1, 1), (2, 2), "b"), None),
+                    ],
+                    unmatched_inf=[
+                        ScoredLabeledBoundingBox((8, 8), (9, 9), "b", 0),
+                    ],
+                ),
+            ],
+            CurvePlot(
+                title="F1-Score vs. Confidence Threshold",
+                x_label="Confidence Threshold",
+                y_label="F1-Score",
+                curves=[
+                    Curve(
+                        x=[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                        y=[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                        label="two single class matchings",
+                    ),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+            CurvePlot(
+                title="Precision vs. Recall",
+                x_label="Recall",
+                y_label="Precision",
+                curves=[
+                    Curve(
+                        x=[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                        y=[0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0],
+                        label="two single class matchings",
+                    ),
+                ],
+                x_config=None,
+                y_config=None,
+            ),
+        ),
+        (
+            "large",
+            [
+                MulticlassInferenceMatches(
+                    matched=[
+                        (
+                            LabeledBoundingBox(top_left=(1.0, 1.0), bottom_right=(6.0, 6.0), label="cow"),
+                            ScoredLabeledBoundingBox((1.0, 1.0), (6.0, 6.0), "cow", 0.9),
+                        ),
+                        (
+                            LabeledBoundingBox(top_left=(10.0, 10.0), bottom_right=(22.0, 22.0), label="cow"),
+                            ScoredLabeledBoundingBox((10.0, 10.0), (20.0, 20.0), "cow", 0.9),
+                        ),
+                    ],
+                    unmatched_gt=[
+                        (
+                            LabeledPolygon(
+                                points=[(1, 1), (1, 2), (2, 2), (2, 1)],
+                                label="cow",
+                            ),
+                            ScoredLabeledPolygon(
```

Comment:
Shouldn't this be also added to `unmatched_inf`?  
Currently this pair is only counted as 1 FN, but shouldn't this be 1 FN and 1 FP?